qwandaq/src/main/java/life/genny/qwandaq/managers/CacheManager.java

```diff
@@ -252,7 +252,25 @@ public List<BaseEntity> getBaseEntitiesByPrefix(String cacheName, String prefix)
 	}
 
 	/**
-	 * Get the max entity  id.
+	 * Get the max attribute id.
+	 *
+	 * @return the max id
+	 */
+	public Long getMaxAttributeId() {
+		QueryFactory queryFactory = Search.getQueryFactory(cache.getRemoteCacheForEntity(GennyConstants.CACHE_NAME_ATTRIBUTE));
+		Query<Attribute> query = queryFactory
+				.create("from life.genny.qwandaq.persistence.attribute.Attribute order by id desc");
+		QueryResult<Attribute> queryResult = query.maxResults(1).execute();
+
+		Attribute attribute = queryResult.list().get(0);
```

Comment:
(edge case) if queryResult is empty then this will be an ArrayIndexOutOfBoundsException. Check if queryResult isEmpty and return 0 if so